app/Http/Controllers/DownloadControllerNew.php

```diff
@@ -64,24 +67,44 @@ public function index (Request $request)
                     $location_id = $country->id;
                 }
             }
+            else
+            {
+                $path = $year.'/'.$month.'/'.$day.'/' . 'Global_OpenLitterMap.csv';
+            }
 
-            /* Dispatch job to create CSV file for export */
-            (new CreateCSVExport($request->type, $location_id))
-                ->queue($path, 's3', null, ['visibility' => 'public'])
-                ->chain([
-                    // These jobs are executed when above is finished.
-                    new EmailUserExportCompleted($email, $path)
-                    // new ....job
-                ]);
+            $this->export($request->type, $location_id, $path, $email);
 
             return ['success' => true];
         }
 
-        catch (\Exception $e)
+        catch (Exception $e)
         {
-            \Log::info(['download failed', $e->getMessage()]);
+            Log::info(['download failed', $e->getMessage()]);
 
             return ['success' => false];
         }
     }
+
+    /**
+     * @param string $type
+     * @param int $locationId
+     * @param string $path
+     * @param string $email
+     */
+    protected function export(string $type, int $locationId, string $path, string $email): void
```

Comment:
What are we exporting here? Any location? Global? Comment needs a title

Reply:
I'm adding a comment on this.